docs/user_guide/fairness_in_machine_learning.rst

```diff
@@ -141,6 +141,94 @@ which groups these values were observed, as well as the difference and ratio
 between the maximum and the minimum values. For more information refer to the
 subpackage :code:`fairlearn.metrics`.
 
+Failure modes encountered when abstracting
+--------------------------------------------------
+
+In "Fairness and Abstraction in Sociotechnical Systems" Selbst et al. [#4]_
+identify failure modes that arise from applying abstraction. They identify
+them as:
+
+* *The Framing Trap*
+
+* *The Portability Trap*
+
+* *The Formalism Trap*
+
+* *The Ripple Effect Trap*
+
+* *The Solutionism Trap*
+
+These traps are inspired by the observation that each of these traps are the
+result of failing to consider the way in with social context and technology
+are interrelated, as well as a deeper understanding of "the social" in order to
+solve problems Ackerman [#5]_.
+
+The Framing Trap
+^^^^^^^^^^^^^^^^
+
+Selbst et al. [#4]_ define this as a "failure to model the entire system over
+which a social criterion, such as fairness, will be enforced".
+
+It is the practice of failing to consider the full picture surrounding sources of
+bias.
+
+Labelling is a risk assessment setting. Risk assessment may be used as a predictive
```

Comment:
Ah, I'm not sure I understand what that first sentence means...

Reply:
Reworded